README.md

```diff
@@ -26,6 +26,34 @@ generate the `.yaml` entries here.
 
 ## Using this data
 
+### Marking specific attributes
+It can be helpful to know which specific code elements of a package are vulnerable and this is done by appending an attribute and list of module paths starting from the top level module of a package to the OSV payload. Eg.  
```

Comment:
Can we mention explicitly this can be set in [ecosystem_specific](https://ossf.github.io/osv-schema/#affectedecosystem_specific-field) (and that this section is defining the format for that)? 

e.g.

> "OSV entries in this database have the following [`ecosystem_specific`](https://ossf.github.io/osv-schema/#affectedecosystem_specific-field) definition: 

```
"ecosystem_specific": {
  "imports": [
    { 
       "attribute": string,
       "modules": [ string ],
    }
  ]
}
```

> "imports" is a JSON array containing the modules and attributes affected by the vulnerability...
> For example, a vulnerability that affects `PIL::ImageFont` can be represented as...

This follows the "informal schema definition" convention of the OSV schema. I've also used "imports" for the top level field name, but there might be better names here.

Reply:
That seems reasonable enough to me. Pushed an update with your changes 👍